### PR TITLE
[13.0][ADD] fleet_vehicle_service_calendar: Scheduling meetings for vehicle services

### PR DESCRIPTION
**Pull request by mamcode**
screenshot-localhost_8099-2020 05 21-19_26_12

screenshot-localhost_8099-2020 05 21-19_26_47

### CODE REVIEW

**Comment by mamcode** [comment]
@ivantodorovich This depends on #3, so it does not pass the test yet.

File: fleet_vehicle_service_calendar/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Scheduling Meetings for Vehicle Services",
+    "summary": "Add a smart button in services to schedule meetings.",
+    "category": "Human Resources/Fleet",
+    "author": "Druidoo, Odoo Community Association (OCA)",
+    "maintainers": ["mamcode", "ivantodorovich"],
+    "development_status": "Stable",
+    "website": "https://www.druidoo.io/",
```
Comment (ivantodorovich) [inline fleet_vehicle_service_calendar/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/fleet",
```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
@mamcode can you rebase on `13.0`, now that fleet_vehicle_service_kanban has been merged?

**Comment by mamcode** [comment]
> @mamcode can you rebase on `13.0`, now that fleet_vehicle_service_kanban has been merged?

Done.

**Review by mymage — APPROVED** [review]
It look good for me. The only comment is that for people like me (with not much knowledge on Odoo) some other comments on the code would be gold ;-)

**Pull request merged**